engine/modules/opentelemetry/src/opentelemetry.cc

```diff
@@ -0,0 +1,279 @@
+/**
+ * Copyright 2024 Centreon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ */
+
+#include "com/centreon/exceptions/msg_fmt.hh"
+
+#include "com/centreon/engine/modules/opentelemetry/opentelemetry.hh"
+
+#include "otl_fmt.hh"
+#include "otl_server.hh"
+
+using namespace com::centreon::engine::modules::opentelemetry;
+
+/**
+ * @brief Construct a new open telemetry::open telemetry object
+ *
+ * @param config_file_path
+ * @param io_context
+ */
+open_telemetry::open_telemetry(
```

Comment:
I'm very perturbed with that: the file name is opentelemetry.cc and the class is open_telemetry.
Could we change this to opentelemetry everywhere please.